src/plugins/console/api_server/es_5_0/mappings.js

```diff
@@ -113,9 +96,13 @@ module.exports = function (api) {
           },
           boost: 1.0,
           null_value: '',
+
+          // norms
           omit_norms: {
             __one_of: [true, false]
           },
+          norms: BOOLEAN,
```

Comment:
Checked with ES team. `omit_norms` and `norms` are both valid in ES 5.0 but `omit_norms` is deprecated. So I have removed it as a suggestion in Console.